src/runtime/app.config.ts

```diff
@@ -349,7 +349,7 @@ const input = {
       xl: 'h-6 w-6'
     },
     leading: {
-      wrapper: 'absolute inset-y-0 left-0 flex items-center',
+      wrapper: 'absolute inset-y-0 left-0 rtl:right-0 flex items-center',
```

Comment:
Are you sure this will work? On `rtl` mode, this will apply `left-0` and `right-0` here.

Reply:
@HassanKadhim

Reply:
@benjamincanac 
@benjamincanac 

I checkd it, and changed to `start-0`

After :
❌
![image](https://github.com/nuxtlabs/ui/assets/37311945/f96bbf9f-231c-4ca2-b450-88105e701bfb)

Before: 
✅
![image](https://github.com/nuxtlabs/ui/assets/37311945/901b701a-cf08-4b3c-9921-00cd5bb5b886)

Reply:
Yes, its works

Reply:
Yes, its works

Reply:
Yes, its works